mkdocs.yml

```diff
@@ -23,6 +23,8 @@ nav:
       - Identity: 'identity.md'
       - Trust issues: 'trust.md'
     - Demos:
+      - Aries Cloud Agents Example: 'aries-cloud-agent-example.md'
```

Comment:
remove the example, it is listed further below in a different section

Reply:
Done